index.bs

```diff
@@ -1394,21 +1404,38 @@ To <dfn>parse filter data</dfn> given a |value|:
 Issue: Determine whether to limit [=string/length=] or
 [=string/code point length=] for |filter| and |d| above.
 
+To <dfn>parse filter config</dfn> given a |value|:
+
+1. If |value| is not a [=map=], return null.
+1. Let |lookbackWindow| be null.
+1. If |value|["`lookback_window`"] [=map/exists=]:
```

Comment:
It's worth noting that in the Protected Audiences extending private aggregation explainer, they prefix "special" keys with "reserved." (https://github.com/WICG/turtledove/blob/main/FLEDGE_extended_PA_reporting.md)

it might make sense to do that here, and would probably lead to less compat issues (although  this is probably marginal anyway).

Reply:
The key was updated with the "_" prefix.